src/containers/AppRoutes/AppRoutes.tsx

```diff
@@ -22,7 +21,6 @@ export default function AppRoutes() {
         <Route index element={<Home />} />
         <Route path="/p/:id" element={<PlaylistScreenRouter />} />
         <Route path="/m/:id/*" element={<MediaScreenRouter />} />
-        <Route path="/s/:id/:slug" element={<Series />} />
```

Comment:
I think we may want to keep this routing. I believe it potentially helps with SEO and webcrawlers.

Reply:
It looks like legacy code we left to support smooth transition from the old approach where series had a dedicated url to a new one, where we have `m` url for different cases. That way a customer could be redirected to a new `m` page. However it seems that we are ready to remove it as pretty much time has passed since we implemented it. 
@ChristiaanScheermeijer

Reply:
Hm, not sure I understand. To show series title and description we use a media item now (new flow). We will remove title / description from the series item.

Reply:
You are right. For series we show only title now, but no description.

<img width="1342" alt="Screenshot 2023-04-21 at 11 01 05" src="https://user-images.githubusercontent.com/27254986/233594110-b869b734-4b7a-429b-91f7-582576440596.png">

We can also have a custom screen for Series using `/m/` route (MediaSeries). For now its main goal is more about redirecting, but we can set it up as sort of a landing page.

I just thought that our main concept was to use `/m/` and `/p/` routes for different purposes. So `/s/` route looks here like a part of legacy.

Reply:
Tbh, I like the /s/ route for series. I forget exactly why we decided to redirect it to /m/, but I remember having a discussion about it. I would say unless there's a strong reason to get rid of it as part of this PR, let's leave those routes and the <SeriesRedirect> component as is and make any changes in another PR.

Reply:
Well, it makes sense because series are actually media items stored in playlists. So in order not to build extra conditions when clicking on card - we just use the same `m` route. In case we use `s` route, then we should probably calculate whether a media item is a real series item and also consider two flows...which seems to result in complicated conditions.

How I personally want it to work (HBO example):

1. We have two separate pages: Series and Episode.
2. Series page is a page with a list of all seasons and episodes
3. Episode page is a page of selected episode with other episodes of the same season or just basic episode info. You can go from this page to the Series page. This page can be use in the playlist when a customer wants to highlight some recent episodes (TBN example).

However for now it is a bit complicated to implement it without extra conditional logic because for our old flow we can't get an original mediaId of the dumb item using an episode.

It looks like we should start with just adapting existing old flow for the new approach. Then when we can get rid of the old flow we can also change the behaviour.